test/variable.jl

```diff
@@ -467,6 +467,19 @@ function test_variable_symmetric(ModelType)
     @test y[1, 2] === y[2, 1]
 end
 
+
+function test_variable_skewsymmetric(ModelType)
+    model = ModelType()
+
+    @variable model x[1:2, 1:2] in SkewSymMatrixSpace()
+    @test x[1, 2] == -x[2, 1]
+    @test x[1, 1] == 0
+    @test x[2, 2] == 0
+    @test model[:x] === x
+
+    y = @variable model [1:2, 1:2] in SkewSymMatrixSpace()
+    @test y[1, 2] == -y[2, 1]
```

Comment:
I would add a 3 x 3 example as the 2 x 2 examples only has 1 non-diag entry so it may not cover all corner cases